iidm/iidm-impl/src/test/java/com/powsybl/iidm/network/impl/VariantManagerImplTest.java

```diff
@@ -149,4 +150,25 @@ public void test() {
         assertEquals(Collections.singleton(0), variantManager.getVariantIndexes());
         assertEquals(2, identifiable1.reducedCount);
     }
+
+    @Test
+    public void testMultipleNetworks() {
+        ObjectStore objectStore = new ObjectStore();
```

Comment:
Recently, the ObjectStore class has been renamed to NetworkIndex. Could you fix this?

Reply:
Fixed after rebasing.